<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>writemail</class>
 <widget class="QWidget" name="writemail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;Sans Serif&quot;;
color: rgba(100, 100, 100,100);</string>
   </property>
   <property name="text">
    <string>WRITE MAIL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(185, 185, 185, 150);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SENDB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:0px;
font: 11pt &quot;Liberation Mono&quot;;
background-color: rgba(20, 20, 20, 30);
}

QPushButton:hover{
	background-color: rgba(143, 143, 143, 100);
	color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD ATTACHMENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SAVEB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:0px;
font: 11pt &quot;Liberation Mono&quot;;
background-color: rgba(20, 20, 20, 30);
}

QPushButton:hover{
	background-color: rgba(143, 143, 143, 100);
	color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CLOSEB">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:0px;
font: 11pt &quot;Liberation Mono&quot;;
background-color: rgba(20, 20, 20, 30);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgba(143, 143, 143, 100);
}
</string>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>701</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TEXT</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>标题      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>地址      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
